notification-portlet-webapp/src/main/java/org/jasig/portlet/notice/controller/rest/NotificationRestV2Controller.java

```diff
@@ -75,6 +75,8 @@
     public List<NotificationEntry> fetchNotifications(HttpServletRequest request) {
         final NotificationResponse response = repository.fetch(request);
         final List<NotificationEntry> rslt = notificationResponseFlattener.flatten(response);
+        logger.debug("rslt count = {}", rslt.size());
```

Comment:
For logs, full words often are easier to process
```suggestion
        logger.debug("result count = {}", rslt.size());
```

Reply:
Good catch!